centos-7.template

```diff
@@ -138,6 +138,9 @@ ExecStart=/usr/bin/docker daemon -H tcp://0.0.0.0:2376 \
                        --add-runtime docker-runc=/usr/libexec/docker/docker-runc-current \
                        --default-runtime=docker-runc \
                        --storage-driver devicemapper \
+                       --storage-opt dm.fs=xfs \
+                       --storage-opt dm.thinpooldev=/dev/mapper/minishift_vg-docker--pool \
+                       --storage-opt dm.use_deferred_removal=true \
```

Comment:
Hmm..., so looks like equivalent of /etc/sysconfig/docker-storage you have hardcoded here.

But /etc/sysconfig/docker-storage is still important. Using that file docker-storage-setup decides if it has already setup a thin pool and it does not have to do anything or it needs to create a new thin pool. 

As of now docker-storage-setup behavior is that it recreates the /etc/sysonfig/docker-storage if lvm thin pool already exists but there are no guarantees that this behavior will continue.

If dss has generated /etc/sysconfig/docker-storage file in the past it should be available to it on subsequent invocations.